canonical-websites/snapcraft.io
# d3 version of account metrics

# Done

Implement the metrics graph in d3.

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account/snaps/{snap_name}/metrics
- Make sure the graph works as close to what's currently in production
